src/nvim/os/env.c

```diff
@@ -667,7 +667,7 @@ void expand_env_esc(char *restrict srcp, char *restrict dst, int dstlen, bool es
         int c = dstlen - 1;
         while (c-- > 0
                && *tail
-               && vim_isfilec(*tail)
+               && vim_isfilec((uint8_t)(*tail))
```

Comment:
This one is impossible to test on CI as it requires a Unicode user name.

Reply:
Can't we just add a user as part of CI setup?

Reply:
I thought of this after posting this comment, but it sounds like an overkill.